### DISCUSSION TITLE
Errored PR's in Dependency Dashboard

### DISCUSSION THREAD

**Discussion by victor-xian** [renovatebot/renovate#16066] [Request Help] [answered]
Running self-hosted Renovate Bot as Github App on Github Enterprise.

Initial PR's created no problem and Dependency Dashboard shows Open PR's.

After next run of Renovate Bot, Dependency Dashboard shows all Open PR's as "Errored".

Checking debug logs, I got following errors:

```
DEBUG: Pull request creation error (repository=victor-xian/xxxxxxxxxxx, branch=renovate/aws-4.x)
       "err": {
         "name": "HTTPError",
         "code": "ERR_NON_2XX_3XX_RESPONSE",
         "timings": {
           "start": 1655205936764,
           "socket": 1655205936764,
           "lookup": 1655205936768,
           "connect": 1655205936768,
           "secureConnect": 1655205936770,
           "upload": 1655205936770,
           "response": 1655205936908,
           "end": 1655205936908,
           "phases": {
             "wait": 0,
             "dns": 4,
             "tcp": 0,
             "tls": 2,
             "request": 0,
             "firstByte": 138,
             "download": 0,
             "total": 144
           }
         },
         "message": "Response code 422 (Unprocessable Entity)",
         "stack": "HTTPError: Response code 422 (Unprocessable Entity)\n    at Request.<anonymous> (/usr/src/app/node_modules/got/dist/source/as-promise/index.js:118:42)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)",

```
Looks like Renovate Bot is trying to recreate the same PR even though the PR was already created.

Any idea?

**Comment by rarkins** [comment]
Renovate CLI is not a GitHub app, so please provide more details about exactly how you are running it. Eg do you have your own app wrapper, do you run it as an action, etc?

**Reply by victor-xian** [reply]
@rarkins I created a Github App as RenovateBot and run it in Kubernetes Cronjob authenticated by Github App installation token as per the docs. Hope this clarifies for you.

**Reply by viceice** [reply]
did you configure `username` and `gitAuthor`? They're required when using app token.

https://docs.renovatebot.com/modules/platform/github/#running-as-a-github-app

**Reply by rarkins** [reply]
And if you configured username, make sure you configured it like `renovate[bot]` and not `renovate`

**Reply by victor-xian** [reply]
@rarkins @viceice Yes it's configured as RenovateBot[bot] - RenovateBot is the Github app name.

**Reply by rarkins** [reply]
Try `renovatebot[bot]` in lower case

**Comment by victor-xian** [comment, marked as answer]
@rarkins All lower case works! Thanks a lot. Do you know why the upper case failed?

**Reply by rarkins** [reply]
Because GitHub lower cases app names when converting it to usernames
